packages/bpk-component-popover/src/bpk-popover.scss

```diff
@@ -29,6 +29,11 @@
   outline: 0;
   opacity: 1;
 
+  @include bpk-breakpoint-mobile {
+    max-width: 90%;
+    margin-left: 5%;
```

Comment:
this doesn't look very RTL friendly :)

Reply:
I reckon this could be solved with `margin-right: $bpk-spacing-sm; margin-left: $bpk-spacing-sm;` no max width.

Reply:
Will give it a go! 

And yep I totally forgot about RTL. Maybe we should have a Danger rule that just says "DON'T FORGET ABOUT RTL, SHAUN"

Reply:
@matthewdavidson yep your suggestion worked fine - you're a gentleman and a scholar. Will push the change now.